src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -9,6 +9,7 @@
     <Title>Datadog APM</Title>
     <Description>Manual instrumentation library for Datadog APM</Description>
     <Authors>lucas.pimentel.datadog;bmermet</Authors>
+    <LangVersion>7.1</LangVersion>
```

Comment:
I've been trying to avoid it if possible because it would force users compiling from source to use a newer compiler than they might be using right now. Of course, I broke my own rule here only to use `default` vs `default(T)`, which in retrospect seems like an insignificant benefit.